urls.py

```diff
@@ -61,6 +61,16 @@
     'distro/core/qiime20176-1497566713.zip':
         'https://s3-us-west-2.amazonaws.com/qiime2-data/distro/core/qiime20176-1497566713.zip',
 
+    # 2017.7 DISTRO
+    'distro/core/qiime2-2017.7-conda-osx-64.txt':
+        'https://raw.githubusercontent.com/qiime2/environment-files/master/2017.6/release/qiime2-2017.7-conda-osx-64.txt',
```

Comment:
We can actually preemptively use the release URL, which is what we have [done in the past](https://github.com/qiime2/data.qiime2.org/pull/16/files#diff-de6dd4b4c889fe0882cfd3f6df5aa451R56). Updating now!